src/Sulu/Component/Content/Document/Subscriber/RedirectTypeSubscriber.php

```diff
@@ -28,7 +28,7 @@ public static function getSubscribedEvents()
     {
         return [
             Events::METADATA_LOAD => 'handleMetadataLoad',
-            Events::PERSIST => 'handlePersist',
+            Events::PERSIST => ['handlePersist', 15],
```

Comment:
Add a comment saying why this priority is required.

@wachterjohannes yeah, @danrot already mentioned something similar, I think it is a good idea.